 <launch>
   <node pkg="baxter_tools" type="enable_robot.py" name="enable" args="-e" output="screen"/>
  <node pkg="baxter_trajectory_streamer" type="rt_arm_as" name="rt_arm_as"  output="screen"/>
  <node pkg="baxter_trajectory_streamer" type="left_arm_as" name="left_arm_as"  output="screen"/> 
  <node pkg="cartesian_planner" type="baxter_cart_move_as" name="baxter_cart_move_as" output="screen"/>
  <node pkg="rviz" type="rviz" name="rviz" args="-d $(find coordinator)/rviz/mobile_manip.rviz"/> 
  <node pkg="baxter_playfile_nodes" type="baxter_playback" name="baxter_playback" args="$(find baxter_playfile_nodes)/pre_pose_right.jsp $(find baxter_playfile_nodes)/pre_pose_left.jsp" output="screen"/> 
  <node pkg="baxter_head_pan" type="baxter_head_pan_zero" name="baxter_head_pan_zero"  output="screen"/>    
  <node pkg="baxter_playfile_nodes" type="baxter_playfile_service" name="baxter_playfile_service"  output="screen"/>  
  <node pkg="example_rviz_marker" type="triad_display" name="triad_display"/>
  <node pkg="object_grabber" type="object_grabber_as" name="object_grabber_as"  output="screen"/>
  <node pkg="object_finder" type="object_finder_as" name="object_finder_as"  output="screen"/>  
  <node pkg="coordinator" type="command_bundler" name="command_bundler"  output="screen"/>
  <node pkg="example_gazebo_set_state" type="set_block_state" name="set_block_state"  output="screen"/>
  <node pkg="coordinator" type="open_loop_nav_service" name="open_loop_nav_service"  output="screen"/>  
  <node pkg="coordinator" type="open_loop_yaw_service" name="open_loop_yaw_service"  output="screen"/>
  <!-- roslaunch baxter_variations mobot_startup_navstack.launch  -->
  <include file="$(find baxter_variations)/mobot_startup_navstack.launch">
 </include> 
    
</launch>
